hotsos/client.py

```diff
@@ -116,46 +129,41 @@ def minimise(self, summary, mode):
         log.warning("Unknown minimalmode '%s'", mode)
         return summary
 
-    def get(self, fmt='yaml', html_escape=False, minimal_mode=None,
-            plugin=None, max_level=2):
-        if plugin:
-            filtered = {plugin: self._summary[plugin]}
+    def get(self, params: OutputManagerParameters = OutputManagerParameters()):
+        if params.plugin:
+            filtered = {params.plugin: self._summary[params.plugin]}
         else:
             filtered = self._summary
 
-        if minimal_mode:
-            filtered = self.minimise(filtered, minimal_mode)
-
-        if fmt not in self.SUMMARY_FORMATS:
```

Comment:
This is no longer needed as we now enforce fmt to be one of the values in the `SUPPORTED_SUMMARY_FORMATS`